2001-3000.po

```diff
@@ -26,37 +26,37 @@ msgstr "Łóżko Medyczne"
 #. STRINGS.BUILDINGS.PREFABS.MESHTILE.DESC
 msgctxt "STRINGS.BUILDINGS.PREFABS.MESHTILE.DESC"
 msgid "How do Duplicants keep their tiny feet from slipping through the holes?"
-msgstr ""
+msgstr "Jak Duplikanci mogą utrzymywać swoje malutkie stópki na takiej powierzchni?"
 
 #. STRINGS.BUILDINGS.PREFABS.MESHTILE.EFFECT
 msgctxt "STRINGS.BUILDINGS.PREFABS.MESHTILE.EFFECT"
 msgid "Can be used as wall or floor tile to enclose rooms.\n\nDoes not obstruct the flow of <style=\"liquid\">Liquid</style> or <style=\"gas\">Gas</style>."
-msgstr ""
+msgstr "Ta płytka może być zastosowana jako ściana lub podłoga.\n\nNie blokuje przepływu <style=\"liquid\">Cieczy</style> i <style=\"gas\">Gazu</style>."

```

Comment:
U nas w polskim jeśli mamy zdanie zaczynające się od "Nie" to raczej łącznikiem jest "ani", a nie "i" ;)
-> Nie blokuje przepływu Cieczy ani Gazów.